### DISCUSSION TITLE
Welcome to Sotopia!

### DISCUSSION THREAD

**Discussion by XuhuiZhou** [sotopia-lab/sotopia#34] [Announcements]
This is a space for anyone who wants to contribute to the Sotopia projects! Feel free to introduce yourself here and let people know what you are interested in :) 

There's also this **slack** channel `#sotopia-friends`, feel free to join if you are connected to the LTI slack space.

You can view what we are up to [here](https://github.com/orgs/sotopia-lab/discussions/categories/ideas)!

Feel free to join the tasks there and ask relevant questions!

**Comment by XuhuiZhou** [comment]
Hi folks! I am [Xuhui](https://xuhuiz.com/). I am a soon-to-be third-year [LTI](https://www.lti.cs.cmu.edu/) PhD working with [Maarten Sap](https://maartensap.com/).
I am interested in building socially aware AI systems!

**Comment by Algoroxyolo** [comment]
Hi! I am Lorenzo from CMU-Q. Previously did research on Roleplay Agents. Hoping to connect the dots with this project

**Reply by Algoroxyolo** [reply]
Looking at not only the social part but also how can agents be both culturally and emotionally aligned and how cultural and emotions can affect social interaction in llms under sotopia

**Comment by ProKil** [comment]
Hi, I am [Hao](https://zhuhao.me), co-creater of sotopia and PhD candidate. Hope more people get to use [sotopia](https://sotopia.world) in your projects. Feel free to tag me whenever you encounter any problems.

**Comment by CharlieDreemur** [comment]
Hi, I am Charlie and currently doing multi-agent social simulation and policy agent to generate social policy to affect other agents in society, hope to connect with Sotopia Team.

**Reply by XuhuiZhou** [reply]
Welcome @CharlieDreemur! Looking forward to connecting with you :)

**Comment by jane-yrhee** [comment]
Hi everyone, I'm Yeoni, a sophomore at CMU studying Statistics & Machine Learning and AI. I'm excited to contribute to Sotopia, particularly interested in exploring the intersection of social dynamics and AI. Looking forward to learning and collaborating with you all!

**Reply by XuhuiZhou** [reply]
Welcome @yeonir!

**Comment by l-mathur** [comment]
Hello! I'm Leena. I am a PhD student at CMU and a Sotopia collaborator. I am interested in advancing socially-intelligent AI systems, and my research focuses on foundations of multimodal AI, embodied AI, and human-machine interaction.

**Reply by XuhuiZhou** [reply]
Yayyyy welcome @l-mathur

**Comment by lwaekfjlk** [comment]
Hi! I am Haofei. I am currently a master student at CMU LTI and a sotopia collaborator and sotopia-pi creator. I will join UIUC for my PhD in the fall. I am passionate about open-source projects and exciting ideas.

**Comment by JUSTSUJAY** [comment]
Hey Everyone! Sujay here. I am very interested in ML/AI. (esp.research)
Seeing PHD students here surely makes me nervous but I hope to find a way to contribute somehow.
I have previously interned at a stealth startup where we were building a Machine Learning Platform for real-time.
Sometimes I take time to understand and complete some things due to my obsession with being Perfect. 
I am excited to contribute.

**Reply by lwaekfjlk** [reply]
Hi Sujay, nice to meet you. I know Stealth, super cool company.

**Comment by abdibrokhim** [comment]
Hi, a resilient and research-oriented student who embraces every opportunity, regardless of the outcome. Currently I am interning at Dugree as AI researcher. Just now I have found [SOTOPIA-π: Interactive Learning of Socially Intelligent Language Agents](https://arxiv.org/pdf/2403.08715) interesting for me.

**Reply by lwaekfjlk** [reply]
Thanks for your interest!
